### PR TITLE
feat: improve title and layout of the confirm dialog

### PR DESCRIPTION
**Pull request by shuuji3**
This is follow-up PR for #2296.

Actually the confirm dialog component has had both title description so all I had to is just using `description` prop in addition to `title` prop. Currently, long test is used as `title` that was caused centered bold text.

So I moved the previous `title` to `description` and added a new short `title` like `Delete post` instead.

I also tried to implement the button focus but it was not strightfoward so I leave it for the future issue.

## Before
Screenshot 2023-08-02 at 2 05 58

## After
Screenshot 2023-08-02 at 20 01 58

### CODE REVIEW

File: components/status/StatusActionsMore.vue [hunk 1]
```diff
@@ -83,18 +84,12 @@ async function deleteStatus() {
...
   }) !== 'confirm')
     return
 
-  if (process.dev) {
-    // eslint-disable-next-line no-alert
-    const result = confirm('[DEV] Are you sure you want to delete and re-draft this post?')
-    if (!result)
-      return
-  }
-
```
Comment (shuuji3) [inline components/status/StatusActionsMore.vue:97]: We will be asked by confirm dialog with this update so this duplicated alert won't be needed.

File: composables/masto/relationship.ts [hunk 2]
```diff
@@ -41,6 +41,7 @@ export async function toggleFollowAccount(relationship: mastodon.v1.Relationship
   if (unfollow) {
     if (await openConfirmDialog({
       title: i18n.t('confirm.unfollow.title'),
+      description: i18n.t('confirm.unfollow.description', [`@${account.acct}`]),
```
Comment (shuuji3) [inline composables/masto/relationship.ts:44]: Unlike the other confirm dialog, this message didn't have an account name place holder so I added one for a better dialog message.

File: locales/en.json [hunk 3]
```diff
@@ -127,27 +129,32 @@
...
       "confirm": "Show",
-      "title": "Are you sure you want to show boosts from {0}？"
+      "description": "Are you sure you want to show boosts from {0}?",
+      "title": "Show boosts"
     },
     "unfollow": {
       "cancel": "Cancel",
       "confirm": "Unfollow",
-      "title": "Are you sure you want to unfollow?"
+      "description": "Are you sure you want to unfollow {0}?",
```
Comment (shuuji3) [inline locales/en.json:156]: As explained in the above comment, I added `{0}` for `en.json` but couldn't add other languages. Is it OK to keep other messages without `{0}`?

**Comment by userquin** [comment]
don't center the title, it should be in left or right (depending on lang ltr or rtl), check also the message and the buttons alignment:
[RTL Support](https://github.com/elk-zone/elk/blob/main/CONTRIBUTING.md#rtl-support) in contributing guide

**Comment by shuuji3** [comment]
Thanks for the feedback and resource. Deleted the title centering and check the RTL language. Is it the expected layout?

Screenshot 2023-08-08 at 3 06 11

Screenshot 2023-08-07 at 2 35 24

File: components/status/StatusActionsMore.vue (hunk 1 above)
Comment (userquin) [reply components/status/StatusActionsMore.vue:97]: Keep this confirm, it is for dev only

**Comment by userquin** [comment]
LGTM we can add focus and maybe focus trap later, we should add also a11y (aria roles and proper markup)

File: components/status/StatusActionsMore.vue (hunk 1 above)
Comment (shuuji3) [reply components/status/StatusActionsMore.vue]: @userquin All right, reverted this just now.

File: locales/en.json (hunk 3 above)
Comment (userquin) [reply locales/en.json:156]: Can you add a new one entry and use `te` to check if present?

Comment (userquin) [reply locales/en.json:156]: If missing new one use current otherwise use the new one with `{0}`

Comment (userquin) [reply locales/en.json:156]: IIRC `te` is for checking entry.

Comment (userquin) [reply locales/en.json:156]: upps, forgot it, we'll need to add the title, I'll add the title for spanish and try to add focus trap

Comment (userquin) [reply locales/en.json:156]: rn all confirm dialog titles will be using `en`  entry

**Review by userquin — APPROVED** [review]

**Review by ryanccn — APPROVED** [review]

**Comment by patak-dev** [comment]
@shuuji3 this looks good to me, thanks! If you resolve the conflicts, we can merge it.

**Comment by shuuji3** [comment]
@patak-dev OK, I've just resolved the code conflict. Thanks!

**Review by patak-dev — APPROVED** [review]

**Pull request merged**